### PR TITLE
fix: improve TypeScript types

### PR DESCRIPTION
**Pull request by JustinBeckwith**

### CODE REVIEW

File: src/filter.ts [hunk 1]
```diff
@@ -156,7 +160,7 @@ export class Filter {
    * //   startTest2Exclusive: 'value3'
    * // }
    */
-  static createRange(start, end, key) {
+  static createRange(start, end, key: string) {
```
Comment (alexander-fenster) [inline src/filter.ts]: looks like... Pascal. What are the types for `start` and `end`? :)

File: src/filter.ts [hunk 2]
```diff
@@ -894,23 +892,22 @@ export class Filter {
    *   }
    * ];
    */
-  value(value) {
+  value(value): void {
```
Comment (alexander-fenster) [inline src/filter.ts]: missing type for the parameter

File: src/index.ts [hunk 3]
```diff
@@ -917,5 +948,4 @@ promisifyAll(Bigtable, {
 module.exports = Bigtable;
```
Comment (alexander-fenster) [inline src/index.ts]: This is unrelated, but I don't like this :)  We can change this to proper TypeScript exports.

**Comment by JustinBeckwith** [comment]
Trying to do this bit by bit :) I find the larger the PR the less likely it is to work 😂 But yeah, noImplicitAny coming in like a 5 PR set.

File: src/filter.ts (hunk 1 above)
Comment (JustinBeckwith) [reply src/filter.ts]: Added some cheap well know ones first.  This library is a trip man.  Will come back around for completeness again.

**Pull request merged**